content/modals/US/v2_short_term_xo.json

```
@@ -0,0 +1,50 @@
+{
+    "meta": {
+        "product": "PAY_LATER_SHORT_TERM",
+        "periodicPayment": "{formattedPeriodicPayment}",
+        "minAmount": "{minAmount}",
+        "maxAmount": "{maxAmount}",
+        "qualifying": "{qualifying_offer}",
+        "amount": "{transaction_amount}",
+        "apr": "{apr}",
+        "preapproved": true,
+        "variables": {
+            "transaction_amount": "${eval(transaction_amount ? transaction_amount : '-')}",
+            "qualifying_offer": "${eval(CREDIT_OFFERS_DS.qualifying_offer ? CREDIT_OFFERS_DS.qualifying_offer : 'false')}",
+            "financing_code": "${CREDIT_OFFERS_DS.financing_code}",
+            "formattedPeriodicPayment": "${CREDIT_OFFERS_DS.formattedPeriodicPayment}",
+            "total_payments": "${CREDIT_OFFERS_DS.total_payments}",
+            "formattedMinAmount": "${CREDIT_OFFERS_DS.formattedMinAmount}",
+            "formattedMaxAmount": "${CREDIT_OFFERS_DS.formattedMaxAmount}",
+            "formattedTotalCost": "${CREDIT_OFFERS_DS.formattedTotalCost}",
+            "minAmount": "${CREDIT_OFFERS_DS.minAmount}",
+            "maxAmount": "${CREDIT_OFFERS_DS.maxAmount}",
+            "apr": "${CREDIT_OFFERS_DS.apr}",
+            "nominal_rate": "${CREDIT_OFFERS_DS.nominal_rate}"
+        }
+    },
+    "content": {
+        "headline": "Pay in 4 interest-free payments",
+        "subheadline": "No impact on credit score and no late fees. Available for purchases of {formattedMinAmount} to {formattedMaxAmount}.",
+        "qualifyingSubheadline": "Split your purchase of {formattedTotalCost} into {total_payments} with no impact on credit score and no late fees.",
+        "donutTimestamps": ["Today", "2 weeks", "4 weeks", "6 weeks"],
+        "instructions": {
+            "instructionHeadline": "How to use Pay in 4",
+            "instructionSubHeadline": "Confirm your info and apply, set up autopay, and review your purchase."
+        },
+        "preapproval": {
+            "preapprovalHeadline": "You’re pre-approved",
+            "preapprovalSubHeadline": "Pay {formattedTotalCost} in {total_payments} interest-free payments.",
+            "preapprovalDisclaimerHeadline": "What to know about pre-approval",
+            "preapprovalDisclaimerBody": [
+                "If you no longer meet the criteria used to select you for this offer, or give info we can’t match, pre-approval won’t apply and you may be declined for Pay in 4.",
+                "This offer starts on {current_date} and ends when you exit checkout, or when the session expires, or if you make another Pay in 4 purchase while this session is active.",
+                "We didn't use info from a credit bureau to make this offer."
+            ]
+        },
```

```suggestion
        },
        "cta": {
            "buttonTextEligible": "Continue with Pay in 4",
            "buttonTextIneligible": "Back to Checkout",
            "link": "https://www.paypal.com/link/to/short-term-application"
        },
```
Looks like we're missing the call to action checkout button content similar to what we have in our Pay Monthly checkout modal. We can add it right below the preapproval content.